conda_forge_tick/migrators/version.py

```diff
@@ -195,23 +195,31 @@ def migrate(
     ) -> "MigrationUidTypedDict":
         version = attrs["new_version"]
 
-        with open(os.path.join(recipe_dir, "meta.yaml")) as fp:
-            raw_meta_yaml = fp.read()
+        recipe_dir = Path(recipe_dir)
+        meta_yaml = recipe_dir / "meta.yaml"
+        recipe_yaml = recipe_dir / "recipe.yaml"
+        if meta_yaml.exists():
+            raw_meta_yaml = meta_yaml.read_text()
 
-        updated_meta_yaml, errors = update_version(
-            raw_meta_yaml,
-            version,
-            hash_type=hash_type,
-        )
+            updated_meta_yaml, errors = update_version(
+                raw_meta_yaml,
+                version,
+                hash_type=hash_type,
+            )
 
-        if len(errors) == 0 and updated_meta_yaml is not None:
-            with pushd(recipe_dir):
-                with open("meta.yaml", "w") as fp:
-                    fp.write(updated_meta_yaml)
-                self.set_build_number("meta.yaml")
+            if len(errors) == 0 and updated_meta_yaml is not None:
+                meta_yaml.write_text(updated_meta_yaml)
+                self.set_build_number(meta_yaml)
 
-            return super().migrate(recipe_dir, attrs)
-        else:
+                return super().migrate(recipe_dir, attrs)
```

Comment:
This return clause needs be added for v2 or combined into a single set of statements at the end of the method.

Reply:
thanks, fixed.